__docs__/wonder-blocks-dropdown/single-select.stories.tsx

```diff
@@ -340,6 +343,45 @@ Disabled.parameters = {
     },
 };
 
+/**
+ * This select is in an invalid state.
+ */
+export const Invalid: StoryComponentType = {
+    render: (args) => (
+        <SingleSelect
+            {...args}
+            placeholder="Choose a fruit"
+            onChange={() => {}}
+            selectedValue=""
+            isInvalid={true}
+            testId="singleselect-invalid"
+        >
+            {items}
+        </SingleSelect>
+    ),
+    play: async ({canvasElement}) => {
+        const canvas = within(canvasElement);
+
+        // Get HTML element
+        const dropdown = canvas.getByTestId("singleselect-invalid");
+
+        // Hover style
+        await userEvent.hover(dropdown);
+        expect(dropdown).toHaveStyle("border-color: #d92916");
+        expect(dropdown).toHaveStyle("border-width: 2px");
+
+        // Focus style
+        await fireEvent.focus(dropdown);
+        expect(dropdown).toHaveStyle("border-color: #d92916");
+        expect(dropdown).toHaveStyle("border-width: 2px");
+
+        // Active (mouse down) style
+        // eslint-disable-next-line testing-library/prefer-user-event
+        await fireEvent.mouseDown(dropdown);
+        expect(dropdown).toHaveStyle("background-color: rgb(243, 187, 180)");
+    },
```

Comment:
**suggestion:** I would suggest to move this into a Jest unit test. I don't think these kind of examples require Storybook interaction tests mostly because the visuals for this can be snapshotted and confirmed in Chromatic (at least the `resting` state). For the other states, I think this can be moved to Jest.

Reply:
ahh i see. i thought interaction tests were more for visual styles but i guess chromatic snapshots makes sense. i'll update it.